docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -0,0 +1,180 @@
+---
+description: Feature and improvement highlights for Grafana v9.5
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '9.5'
+  - release notes
+title: What's new in Grafana v9.5
+weight: -33
+---
+
+# What’s new in Grafana v9.5
+
+Welcome to Grafana 9.5! We're excited to share some major updates to Grafana's navigation, tons of usability improvements to Alerting, and some promising experiments to help you query your Prometheus metrics. Also, read on to learn about our continued migration from API keys to service accounts, as well as deprecation of plugins that use Angular and a field in the InfluxDB data source.
+
+For more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md). For the specific steps we recommend when you upgrade to v9.5, check out our [Upgrade Guide]({{< relref "../../upgrade-guide/upgrade-v9.5/" >}}).
```

Comment:
@chri2547 - I can't get this link to the upgrade guide to work in my local. I'm not sure what I'm doing wrong here.

Reply:
Hm, tried a couple things, same result. @jdbaldry , what are we doing wrong here? Thank you!

Reply:
Fixed.